dnsvalidator/dnsvalidator.py

```diff
@@ -40,12 +41,20 @@ def resolve():
 valid_servers = []
 responses = {}
 
+from ipaddress import ip_address, IPv4Address
+
+def validIPAddress(IP):
+    try:
+        return "IPv4" if type(ip_address(IP)) is IPv4Address else "IPv6"
```

Comment:
Your validation is incorrect. You validate IPv4 addresses only, other IPs are marked as valid, even when they are invalid.

Here is a better solution:
```py
ipType = type(ip_address(IP))
return ipType is IPv4Address or ipType is IPv6Address
```

Reply:
Thank you @AlexanderMac for the review. I have corrected the validation. Please let me know if I am missing something.